INSTALL.md

```diff
@@ -259,6 +259,17 @@ Synapse via https://packages.matrix.org/debian/. They are available for Debian
 
 ```
 sudo apt install -y lsb-release wget apt-transport-https
+```
+
+for Debian/Ubuntu versions shipping `apt` < 1.6 (Debian < 10 / Ubuntu < 18.04), or
+
+```
+sudo apt install -y lsb-release wget
+```
+
+for Debian 10 / Ubuntu 18.04 and newer, followed by
+
+```
```

Comment:
Hrm. I am unconvinced by this.

My concern is that it makes things much more complicated for users to understand what to do. It used to be a single set of commands that people could follow; now they have to read carefully to understand exactly which commands they have to run.

People are terrible at reading.

At the very least, I'd prefer `sudo apt install -y apt-transport-https` to be an additional step which you have to apply for older systems, but why is it a problem if the `apt-transport-https` is installed?

Reply:
Why install useless packages?

Reply:
I thought I explained that. Because it's much easier to say to a user "here, follow this one set of instructions" than "check your apt version and then copy some bits from here, and some other bits from over there".

Reply:
And then he finds out it's a dummy and deinstalls it again...

Anyway, I'll reword the text...

Reply:
> And then he finds out it's a dummy and deinstalls it again...

well, that's no problem. If the user is smart enough to do that, they are welcome to. 

I do see your point, I just think https://github.com/matrix-org/synapse/blob/2362441544b0dd510cae9321c34e76aca3ad9965/INSTALL.md#matrixorg-packages is an order of magnitude harder to understand than https://github.com/matrix-org/synapse/blob/91a7c5ff6d3a4bc30d61da96e4923d391c6d2ed9/INSTALL.md#matrixorg-packages, and people already find our installation instructions scary.

I think I'm going to revert this for now; if you'd like to suggest improvements, feel free to make a separate PR (or better yet, join `#synapse-dev:matrix.org` to discuss).